cmd/clef/main.go

```diff
@@ -459,6 +489,43 @@ func initialize(c *cli.Context) error {
 	return nil
 }
 
+func listAccounts(c *cli.Context) error {
+	internalApi, err := initInternalApi(c)
+	accs, err := internalApi.ListAccounts(context.Background())
+	if err != nil {
+		return err
+	}
+	if len(accs) == 0 {
+		fmt.Println("\nThe keystore is empty.")
+	}
+	fmt.Println()
+	for _, account := range accs {
+		fmt.Println(account.Address)
+	}
+	return err
+}
+
+func listWallets(c *cli.Context) error{
+	internalApi, err := initInternalApi(c)
+	if err != nil {
+		return err
+	}
+	wallets := internalApi.ListWallets()
+	if len(wallets) == 0 {
+		fmt.Println("\nThere are no wallets.")
+	}
+	fmt.Println()
+	for i, wallet := range wallets {
+		fmt.Printf("Wallet %d at %v (%v %v)\n", i, wallet.URL, wallet.Status, wallet.Failure)
+		fmt.Printf("Accounts in Wallet %d:\n", i)
+		for j, acc := range wallet.Accounts {
+			fmt.Printf("Account %d: %v (%v)\n", j, acc.Address, acc.URL)
```

Comment:
```suggestion
			fmt.Printf("  -Account %d: %v (%v)\n", j, acc.Address, acc.URL)
```
This looks as follows in the console:

```
- Wallet 1 at keystore:///home/joe/Code/go-ethereum/testdata/keystore/UTC--2022-11-01T17-05-11.100536003Z--8ef15919f852a8034688a71d8b57ab0187364009 (Locked )
  - Account 0: 0x8Ef15919F852A8034688a71d8b57Ab0187364009 (keystore:///home/joe/Code/go-ethereum/testdata/keystore/UTC--2022-11-01T17-05-11.100536003Z--8ef15919f852a8034688a71d8b57ab0187364009)

- Wallet 2 at keystore:///home/joe/Code/go-ethereum/testdata/keystore/UTC--2022-11-02T11-40-55.527987625Z--99a94ac1168b301449970310904fc38c7bf55037 (Locked )
  - Account 0: 0x99A94ac1168B301449970310904fc38c7Bf55037 (keystore:///home/joe/Code/go-ethereum/testdata/keystore/UTC--2022-11-02T11-40-55.527987625Z--99a94ac1168b301449970310904fc38c7bf55037)
```

and as follows when rendered as markdown:

- Wallet 1 at keystore:///home/joe/Code/go-ethereum/testdata/keystore/UTC--2022-11-01T17-05-11.100536003Z--8ef15919f852a8034688a71d8b57ab0187364009 (Locked )
  - Account 0: 0x8Ef15919F852A8034688a71d8b57Ab0187364009 (keystore:///home/joe/Code/go-ethereum/testdata/keystore/UTC--2022-11-01T17-05-11.100536003Z--8ef15919f852a8034688a71d8b57ab0187364009)